### DISCUSSION TITLE
No need to announce you have submitted your pre-proposal

### DISCUSSION THREAD

**Discussion by hmacdope** [MDAnalysis/mdanalysis#4515] [GSoC Discussions]
Hi all,

No need to tell us on the discussion list that you have submitted your pre-proposal. We will get back to you as soon as we can with a response. 

Cheers, 

Hugo

**Comment by Gunjan-Goyal** [comment]
Hey, I am Gunjan Goyal, contributor at GSoC. I tried joining the chat from MDAnalysis's GSoC page but it shows an error. It would be great if you could help regarding this.
